### PR TITLE
Add Plot.ly Graph to bottom of Submissions view

### PR DESCRIPTION
**Pull request by rishter**
Screen Shot 2019-03-29 at 1 17 13 PM

### CODE REVIEW

File: lib/swbatnet_web/templates/submission/index.html.eex [hunk 1]
```diff
@@ -24,3 +24,13 @@
     <% end %>
   </thead>
 </table>
+
+<div id="graph"></div>
+
+<script>
```
Comment (alexgriff) [inline lib/swbatnet_web/templates/submission/index.html.eex:30]: Can we move this to another file that can `import { App }` and run `makePlot` after the DOM content has loaded.

Just worried about ever being able to find this again if someone else wants to play around with it having it buried in the view like this

**Review by alexgriff — CHANGES_REQUESTED** [review]
Pending the one comment about moving the JS out of the script tag, good to go after that.

Thanks this is exciting!

**Comment by alexgriff** [comment]
Page-specific js in phoenix https://stackoverflow.com/a/38169882

This example is way overengineered for this use case, but our version could be to add a conditional: is a certain dom node on the page, if so run the code to add the chart
https://medium.com/@hoang_nguyen/organize-and-serve-page-specific-javascript-in-phoenix-with-webpack-47a24d169c03

**Comment by rishter** [comment]
miss u @alexgriff

**Pull request closed without merging**